crates/secret-service-proto/src/v1/wire.rs

```diff
@@ -156,28 +142,28 @@ pub enum ServerMessage {
     /// [`WotsSigner::get_128_secret_key`](super::traits::WotsSigner::get_128_secret_key).
     WotsGet128SecretKey {
         /// A set of 20 byte keys, one for each bit that is committed to.
-        key: [u8; 20 * 36],
+        key: WotsKey<36>,
     },
 
     /// Response for
     /// [`WotsSigner::get_256_secret_key`](super::traits::WotsSigner::get_256_secret_key).
     WotsGet256SecretKey {
         /// A set of 20 byte keys, one for each bit that is committed to.
-        key: [u8; 20 * 68],
+        key: WotsKey<68>,
     },
 
     /// Response for
     /// [`WotsSigner::get_128_public_key`](super::traits::WotsSigner::get_128_public_key).
     WotsGet128PublicKey {
         /// A set of 20 byte keys, one for each bit that is committed to.
-        key: [u8; 20 * 36],
+        key: WotsKey<36>,
     },
 
     /// Response for
     /// [`WotsSigner::get_256_public_key`](super::traits::WotsSigner::get_256_public_key).
     WotsGet256PublicKey {
         /// A set of 20 byte keys, one for each bit that is committed to.
-        key: [u8; 20 * 68],
+        key: WotsKey<68>,
```

Comment:
Magic numbers `36` and `68` can be `const`s with docstrings. Or you can try to import one of the multiple WOTS message length constants somewhere in this codebase.
Or maybe we should define these `const`s in `strata-bridge-primitives`?

Reply:
I honestly don't know. There are so many definitions of WOTS keys: `strata-p2p`, `BitVM`, `secret-service` and `strata-bridge`.